storage/merkle/tree.go

```diff
@@ -24,6 +20,13 @@ var (
 // efficient. In summary, we are able to represent key length of up to 65536 bits, i.e. 8192 bytes.
 const maxKeyLength = 8192
 
+var EmptyTreeRootHash []byte
+
+func init() {
+	h, _ := blake2b.New256([]byte{})
+	EmptyTreeRootHash = h.Sum(nil)
+}
+
```

Comment:
* for the test we use a static value (fast)
* but for the run time, we are working with an initialization logic (slower)

for performance reasons, how about we switch the logic:
```suggestion
var EmptyTreeRootHash []byte = []byte{
	14, 87, 81, 192, 38, 229, 67,
	178, 232, 171, 46, 176, 96, 153,
	218, 161, 209, 229, 223, 71, 119,
	143, 119, 135, 250, 171, 69, 205,
	241, 47, 227, 168}
```

and place the following computation in the test (?)
```golang
	h, _ := blake2b.New256([]byte{})
	EmptyTreeRootHash = h.Sum(nil)
```

Reply:
we could but init would only run once during life time of EN